src/Explorer/Panes/SettingsPane/index.tsx

```diff
@@ -0,0 +1,408 @@
+                          Gets or sets the number of concurrent operations run client side during parallel query
+                          execution. A positive property value limits the number of concurrent operations to the set
+                          value. If it is set to less than 0, the system automatically decides the number of concurrent
+                          operations to run.
+                        </Tooltip>
+                      </div>
+
+                      <input
+                        type="number"
+                        required
+                        min={-1}
+                        step={1}
+                        className="textfontclr collid"
+                        role="textbox"
+                        tabIndex={0}
+                        id="max-degree"
+                        ref={maxDegreeRef}
+                        value={maxDegreeOfParallelism}
+                        onChange={(e) => setMaxDegreeOfParallelism(parseInt(e.target.value) || maxDegreeOfParallelism)}
+                        aria-label="Max degree of parallelism"
+                        autoFocus
+                      />
+                    </div>
+                  </div>
+                )}
+                {shouldShowGraphAutoVizOption && (
+                  <div className="settingsSection">
+                    <div className="settingsSectionPart">
+                      <div className="settingsSectionLabel">
+                        Display Gremlin query results as:&nbsp;
+                        <Tooltip>
+                          Select Graph to automatically visualize the query results as a Graph or JSON to display the
+                          results as JSON.
+                        </Tooltip>
+                      </div>
+                      <div className="tabs" role="radiogroup" aria-label="Graph Auto-visualization">
+                        <div className="tab">
+                          <input
+                            type="radio"
+                            id="graphAutoVizOn"
+                            name="graphAutoVizOption"
+                            defaultValue="false"
+                            checked={Boolean(graphAutoVizDisabled)}
+                            onChange={() =>
+                              setGraphAutoVizDisabled(graphAutoVizDisabled === "false" ? "true" : "false")
+                            }
+                          />
+                          <label
+                            htmlFor="graphAutoVizOn"
+                            id="graph-display"
+                            tabIndex={0}
+                            ref={displayQueryRef}
+                            role="radio"
+                            aria-checked={graphAutoVizDisabled === "false"}
+                            onKeyPress={onGraphDisplayResultsKeyDown}
+                          >
+                            Graph
+                          </label>
```

Comment:
Anywhere we are using inputs we should attempt to use the fluent UI components: https://developer.microsoft.com/en-us/fluentui